src/snekmate/auth/mocks/Ownable2StepMock.vy

```diff
@@ -0,0 +1,50 @@
+# pragma version ~=0.4.0b5
+"""
+@title Ownable2Step Module Reference Implementation
+@custom:contract-name Ownable2StepMock
+@license GNU Affero General Public License v3.0 only
+@author pcaversaccio
+"""
+
+
+# @dev We import and initialise the `Ownable` module.
+from .. import Ownable as ow
+initializes: ow
+
+
+# @dev We import and initialise the `Ownable2Step` module.
+from .. import Ownable2Step as o2
+initializes: o2[ownable := ow]
+
+
+# @dev We export (i.e. the runtime bytecode exposes these
+# functions externally, allowing them to be called using
+# the ABI encoding specification) all `external` functions
+# from the `Ownable2Step` module.
+# @notice Please note that you must always also export (if
+# required by the contract logic) `public` declared `constant`,
+# `immutable`, and state variables, for which Vyper automatically
+# generates an `external` getter function for the variable.
+exports: (
+    ow.owner,
```

Comment:
Waiting for https://github.com/vyperlang/vyper/pull/3888 to be merged, and after this will be changed to:

```diff
- ow.owner
+ o2.owner
```

Reply:
Yeeeah :) implemented via https://github.com/pcaversaccio/snekmate/pull/219/commits/d5f20b7c29bc700a208f80baaea8279747cdd676.